src/functions/delimsizing.js

```diff
@@ -124,6 +124,9 @@ defineFunction({
             node.setAttribute("fence", "false");
         }
 
+        node.setAttribute("minsize", delimiter.sizeToMaxHeight[group.size] + "em");
+        node.setAttribute("maxsize", delimiter.sizeToMaxHeight[group.size] + "em");
```

Comment:
[MDN docs](https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mo) suggest `minsize` and `maxsize` specifies the size only when `stretchy` is `true`. Is `stretchy` true by default?

Reply:
No. The default value of stretchy is false. I'll add stretchy = true.

Reply:
I wonder why it did work without it.

Reply:
Browsers are to render a lot of things that don't pass a W3C validation. I suppose this is just one of those cases.